docs/reference/remap/functions/parse_glog.cue

```diff
@@ -0,0 +1,36 @@
+package metadata
+
+remap: functions: parse_glog: {
+	category: "Parse"
+	description: """
+		Parses the `value` using the [glog (Google Logging Library)](https://github.com/google/glog) format.
```

Comment:
```suggestion
		Parses the `value` using the [glog (Google Logging Library)](\(urls.glog)) format.
```

Reply:
Then we typically add a line to `urls.cue`:

```
glog: "https://github.com/google/glog"
```

This helps us maintain URLs and check that they are still working.